### ISSUE TITLE
Download of Go toolchain fails because of redirects

### ISSUE THREAD

**Issue by bkabrda** [sourcegraph/srclib#250]
Downloading Go toolchain has starting failing for me recently:
```
Downloading Go toolchain
Cloning into 'srclib-go'...
fatal: repository 'https://sourcegraph.com/sourcegraph/srclib-go/' not found
failed to install/upgrade Go (sourcegraph.com/sourcegraph/srclib-go) toolchain: exit status 128
```
The problem is that git client can't handle redirects, but the above url redirects:
```
$ curl -I sourcegraph.com/sourcegraph/srclib-go
HTTP/1.1 301 Moved Permanently
```
The redirect target is https://sourcegraph.com/github.com/sourcegraph/srclib.
Everything was working several days ago, so maybe you just setup the redirects recently? Or am I doing something wrong?
Thanks

**Comment by shurcooL** [comment]
Thanks for reporting this. It's a configuration change we've done our side recently that caused this, we'll work on fixing it. /cc @pararthshah

The redirect target should be https://sourcegraph.com/github.com/sourcegraph/srclib-go and that would work as a `git clone` target for now.

**Comment by guilhermebr** [comment]
:+1:

**Comment by pararthshah** [comment]
This is fixed by commit [f37ecd8](https://github.com/sourcegraph/srclib/commit/f37ecd88157b8a4606035ca28ceddc2a09d4ef99), as now all toolchains are fetched directly from GitHub.
